controllers/controlplane/kopscontrolplane_controller.go

```diff
@@ -567,11 +588,96 @@ func (r *KopsControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	}
 	reconciler.awsCredentials = *awsCredentials
 
+	kopsClientset, err := reconciler.GetKopsClientSetFactory(kopsControlPlane.Spec.KopsClusterSpec.ConfigBase)
+	if err != nil {
+		reconciler.Recorder.Eventf(kopsControlPlane, corev1.EventTypeWarning, "FailedInstantiateKopsClient", "failed to instantiate Kops client: %s", err)
+		return resultError, err
+	}
+
+	kopsCluster := &kopsapi.Cluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: owner.GetName(),
+		},
+		Spec: kopsControlPlane.Spec.KopsClusterSpec,
```

Comment:
Does the ownerReference have any use besides indicating the ownership? I thought the relationship was already well indicated by the ClusterRef, and seems like it makes deletion more complicated (since it's a "two-way" relationship)

Besides that, LGTM